cdisc-org/cdisc-rules-engine
# Move dummy data service

This PR fixes the issue `ImportError: cannot import name 'DummyDataService'` when installing the python module.

Steps to test:

1. Clone and build a wheel from this branch: `python -m build`
2. Create a new virtual environment
3. Install the new build in the virtual environment: `pip install .\dist\cdisc_rules_engine-1.0.0-py3-none-any.whl`
4. Verify you can import Dummy Data Service in the virtual environment
